input/fsh/DEF_SearchParameters.fsh

```diff
@@ -19,13 +20,14 @@ Title: "Search by _id in Patients with multipleOr"
 * base[0] = #Patient
 * type = #token
 * multipleOr = true
-* expression = "Resource.id"
-* xpath = "f:Resource/f:id"
+* expression = "Patient.id"
```

Comment:
Do the search parameters show up somewhere in the IG? Where these copied from mCODE or why did we add search parameters for Patient and Condition to CodeX RT?

Reply:
I don't know.   We can delete them, or correct them.

Reply:
https://build.fhir.org/ig/HL7/codex-radiation-therapy/SearchParameter-Patient-id.html
https://build.fhir.org/ig/HL7/codex-radiation-therapy/SearchParameter-Condition-code.html
They appear to be carryovers form mCode. 
https://hl7.org/fhir/us/mcode/STU3/artifacts.html#behavior-search-parameters which includes:
https://hl7.org/fhir/us/mcode/STU3/SearchParameter-Condition-code.html. but not the Patient-id search, since that was redundant with USCore Patient.

Reply:
I think we should remove them. They were never discussed and agreed in CodeX RT context. If they are just inherited, no need to repeat.

Reply:
Deleted